docs/DeveloperGuide.adoc

```diff
@@ -294,72 +296,107 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a(n) ... |I want to ... |So that I can...
+|`* * *` |social student |keep track of who owes me money & how much | not have anyone owe me any money.
 
-|`* * *` |user |add a new person |
+|`* * *` |busy student |keep track of all the tasks I have done | work on tasks that I have yet to do.
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |indecisive student |roll back and forth changes that I have done | track my ever-changing schedule.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |NUS student | view all the prerequisites for a specified module | plan my academic roadmap accordingly.
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |busy student |be reminded when I am nearing a deadline |be on top of all my assignments
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`*` |student |calculate my CAP easily |keep track of my progress in university.
 |=======================================================================
 
-_{More to be added}_
-
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
-
+(For all use cases below, the *System* is the `JARVIS` and the *Actor* is the `user`, unless specified otherwise)
 [discrete]
-=== Use case: Delete person
+=== Use case: Set tabs in finance tracker
```

Comment:
it does not seem clear that the use case is about setting tabs when the 

meaning of 'tabs' is unclear, at first I assumed tabs were more like tabs in a window or browser.